subsys/usb/class/bt_h4.c

```diff
@@ -0,0 +1,314 @@
+/*
```

Comment:
Is this the same as the class/bluetooth, except it does not have dynamic h4/h2 handling?

Reply:
This is using a dedicated PID so it doesn't use the startand bluetooth class.